message_definitions/v1.0/ras_a.xml

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0"?>
 <mavlink>
+  <!-- Contact info: ? -->
+  <!-- URL: ? -->
+  <!-- message ID range: 51000 - 52000 -->
+  <!-- command ID range: 51000 - 52000 -->
```

Comment:
FYI, I have allocated message IDs and command IDs to have the same range to make management a little easier. If this ends up in mavlink/mavlink then you'd need to add contact info and URL - essentially so we know who to contact for changes.

If these changes are made then I'll add a note in mavlink/mavlink to reserve the range.

Reply:
@hamishwillee please do reserve the range in `mavlink/mavlink`. Thanks!

Reply:
> add contact info and URL - essentially so we know who to contact for changes.

Feel free to add me:

```xml
<!-- Contact: Nuno Marques <n.marques21@hotmail.com> | Github: TSC21 -->
```